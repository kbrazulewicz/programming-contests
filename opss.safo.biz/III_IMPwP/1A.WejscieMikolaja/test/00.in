6 150
180 600 600
120 130 140
150 155 400
135 140 500
120 150 650
140 200 145
