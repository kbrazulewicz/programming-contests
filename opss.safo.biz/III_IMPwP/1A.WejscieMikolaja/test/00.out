komin
brak
okno
drzwi
drzwi
okno
